cmd/dfget/app/server.go

```diff
@@ -63,8 +64,16 @@ func initServerFlags() {
 		"be verbose")
 }
 
+func initCfgFromEnvs() {
+	if ip := os.Getenv("LOCAL_IP"); len(ip) > 0 {
```

Comment:
Is there any place to tell users that setting this environment variable in documents? 
Otherwise, it is not so user-friendly, just in code, not in doc.

Reply:
I will update related docs

Reply:
I will update related docs